### PR TITLE
Adds cert-exporter chart

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/3913

### CODE REVIEW

File: pkg/v6/resource/chartconfig/create.go [hunk 1]
```diff
@@ -26,9 +26,11 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		}
 
 		for _, chartConfigToCreate := range chartConfigsToCreate {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chartconfig %s", chartConfigToCreate.Name))
```
Comment (JosephSalisbury) [inline pkg/v6/resource/chartconfig/create.go]: I was going mad with some debugging, this really helped. Happy to remove, but I found it useful.

Comment (kopiczko) [reply pkg/v6/resource/chartconfig/create.go]: We should also have corresponding `created chartconfig` log. See https://github.com/giantswarm/fmt/blob/master/go/log_messages.md

File: pkg/v6/resource/chartconfig/create.go [hunk 2]
```diff
@@ -26,9 +26,11 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
...
+
 			_, err := guestG8sClient.CoreV1alpha1().ChartConfigs(resourceNamespace).Create(chartConfigToCreate)
 			if apierrors.IsAlreadyExists(err) {
-				// fall through
```
Comment (kopiczko) [inline pkg/v6/resource/chartconfig/create.go]: Here would be good to have `did not create ...` log (described in the guide linked in other comment).

File: pkg/v6/resource/chartconfig/create.go [hunk 3]
```diff
@@ -26,9 +26,11 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
...
+				r.logger.LogCtx(ctx, "level", "debug", "message", "chartconfig already exists")
 			} else if guest.IsAPINotAvailable(err) {
```
Comment (kopiczko) [inline pkg/v6/resource/chartconfig/create.go]: Same about  `did not create ...` log.

**Comment by JosephSalisbury** [comment]
thanks to @kopiczko for helping me *supercharge* the logging :D

**Comment by JosephSalisbury** [comment]
```
$ kubectl -n=kube-system get pod -l=app=cert-exporter
NAME                  READY     STATUS    RESTARTS   AGE
cert-exporter-j82z5   1/1       Running   0          2m
cert-exporter-jcdbm   1/1       Running   0          2m
cert-exporter-t9d8l   1/1       Running   0          2m
cert-exporter-x7b82   1/1       Running   0          2m
```
bam!

**Pull request merged**